ktor-client/ktor-client-features/ktor-client-json/ktor-client-serialization/common/src/io/ktor/client/features/json/serializer/KotlinxSerializer.kt

```diff
@@ -81,33 +36,44 @@ class KotlinxSerializer(
         val mapper = deserializationStrategy ?: (type.kotlinType?.let { serializer(it) } ?: type.type.serializer())
         return json.parse(mapper, text)!!
     }
+
+    public companion object {
+        /**
+         * Default [Json] configuration for [KotlinxSerializer].
+         */
+        val DefaultJsonConfiguration: JsonConfiguration = JsonConfiguration(
+            isLenient = true,
```

Comment:
I'd rather not to make `lenient` default choice.
Fail-fast is much better for timely bug-spotting